src/app/support/crowdfundingRewardsSection.tsx

```diff
@@ -50,40 +51,42 @@ export default function CrowdfundingRewardsSection() {
           ))}
         </InfoBox>
       </div>
-      <h1 className="RewardsSectionTitle mb-12">our rewards breakdown:</h1>
+      <h1 className="RewardsSectionTitle pb-12 pt-12">
+        our rewards breakdown:
+      </h1>
       <div className="RewardsSections flex w-full justify-between gap-10">
-        <div className="IndividualRewards flex flex-col items-start gap-6">
-          {/* Why are these buttons? */}
-          <SiteButton
-            aria="for humans"
-            size="large"
+        <div className="IndividualRewards flex flex-col items-start gap-6 py-0">
+          <InfoBox
+            aria="for individuals"
             variant="filled"
             colorScheme="c4"
+            size="small"
+            addClasses="rounded-full self-center"
           >
-            for humans:
-          </SiteButton>
+            for individuals:
+          </InfoBox>
 
           <div className="IndividualRewardsContainer flex flex-col items-start gap-6">
             {printRewardsArray(individualRewardsArray)}
           </div>
         </div>
 
-        <div className="BusinessRewards flex flex-col items-start gap-6 border-l border-midnight/20 pl-10">
-          {/* Why are these buttons? */}
```

Comment:
Even though these are no longer "buttons" they still very much look like buttons. Objectively this is bad UI :(